apollo-gradle-plugin/src/main/kotlin/com/apollographql/apollo/gradle/internal/GUtil.kt

```diff
@@ -0,0 +1,59 @@
+package com.apollographql.apollo.gradle.internal
+
+import java.util.regex.Pattern
+
+/*
+ * Adapted from Gradle GUtil.java so that we can match the task names
+ *
+ * Copyright 2010 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+object GUtil {
+  private val WORD_SEPARATOR: Pattern = Pattern.compile("\\W+")
+
+  fun String.uncapitalize(): String {
+    return if (isNotEmpty() && this[0].isUpperCase()) substring(0, 1).toLowerCase() + substring(1) else this
+  }
```

Comment:
nit: 🤔 I think Kotlin 'String.decapitalize()' provides the same function?

Reply:
:+1: I was looking for `uncaptalize`, I missed `decapitalize`. Thanks for the heads up!